packages/components/src/components/relative-datepicker/relative-datepicker.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from 'react';
+import { toMoment, daysFromTodayTo } from '@deriv/shared/utils/date';
+
+const RelativeDatepicker = props => {
+    const myRef = useRef();
```

Comment:
Please use snake case for the variables. Also, a better naming please (`hidden_input_ref`) comes to mind.

Reply:
Always welcome to suggest to me variable names.